lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -53,8 +53,7 @@ def add_copy_resources_script_phase(native_target)
             test_type = target.test_type_for_product_type(native_target.symbol_type)
             script_path = "${PODS_ROOT}/#{target.copy_resources_script_path_for_test_type(test_type).relative_path_from(target.sandbox.root)}"
             resource_paths = target.all_dependent_targets.flat_map do |dependent_target|
-              include_test_spec_paths = dependent_target == target
-              dependent_target.resource_paths(include_test_spec_paths)
+              dependent_target.resource_paths.values.flatten
```

Comment:
does this need to be uniqued ?

Reply:
same response as above, there were never uniqued